biodatasets/diann_iber_eval/diann_iber_eval.py

```diff
@@ -0,0 +1,330 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+DIANN's corpus consists of a collection of 500 abstracts from Elsevier journal papers
+related to the biomedical domain, where both Spanish and English versions are available,
+with annotations for disabilities appearing in these abstracts.
+"""
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from utils import parsing, schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@proceedings{DBLP:conf/sepln/2018ibereval,
+  author    = {Paolo Rosso and
+               Julio Gonzalo and
+               Raquel Martinez and
+               Soto Montalvo and
+               Jorge Carrillo de Albornoz},
+  title     = {Proceedings of the Third Workshop on Evaluation of Human Language
+               Technologies for Iberian Languages (IberEval 2018) co-located with
+               34th Conference of the Spanish Society for Natural Language Processing
+               {(SEPLN} 2018), Sevilla, Spain, September 18th, 2018},
+  series    = {{CEUR} Workshop Proceedings},
+  publisher = {CEUR-WS.org},
+  volume    = {2150},
+  year      = {2018},
+  url       = {http://ceur-ws.org/Vol-2150},
+  biburl    = {https://dblp.org/rec/conf/sepln/2018ibereval.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+"""
+
+_DATASETNAME = "diann_iber_eval"
+
+_DESCRIPTION = """\
+DIANN's corpus consists of a collection of 500 abstracts from Elsevier journal papers
+related to the biomedical domain, where both Spanish and English versions are available.
+
+This dataset contains annotations for disabilities appearing in these abstracts,
+usually expressed either with a specific word, such as "blindness", or as the limitation
+or lack of a human function, such as "lack of vision".
+
+(Spanish)
+El corpus DIANN se compone de una colección de 500 resúmenes de artículos de revista Elsevier
+del ámbito biomédico, con versiones en español e inglés.
+
+Este conjunto de datos contiene anotaciones para discapacidades que aparecen en dichos resúmenes,
+expresadas por medio de palabras específicas, como "ablepsia", or como la limitación o falta de
+una función, como "falta de visión".
+"""
+
+_HOMEPAGE = "http://nlp.uned.es/diann/"
+
+_LICENSE = "UNKNOWN"
+
+_URLS = {
+    "diannibereval_en": "https://github.com/gildofabregat/DIANN-IBEREVAL-2018/raw/master/DIANN_CORPUS/english.zip",
+    "diannibereval_sp": "https://github.com/gildofabregat/DIANN-IBEREVAL-2018/raw/master/DIANN_CORPUS/spanish.zip",
+}
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION]  # TODO: also for translation?
```

Comment:
yes, adding a translation component would be great and would serve as a very useful example to future participants. the more multi task / schema examples we have the better. In this case, you would add the task to the supported tasks above and define a new `BigBioConfig`. Check out the https://github.com/bigscience-workshop/biomedical/blob/master/examples/muchmore.py for inspiration.

Reply:
Added support for translation in [62496db](https://github.com/bigscience-workshop/biomedical/pull/319/commits/62496db62949f0c86a4b60bdf59c5268dcfe466d).

However, the current tests do not pass for the subset `diannibereval_en` and `diannibereval_es`, because of the missing `bigbio_t2t`. I tried running 

```console
python -m tests.test_bigbio \
    biodatasets/diann_iber_eval/diann_iber_eval.py \
    --subset_id diannibereval_en \
    --schema "KB"
```

but the following persists

```
raise ValueError(f"BuilderConfig {name} not found. Available: {list(self.builder_configs.keys())}")
ValueError: BuilderConfig diannibereval_en_bigbio_t2t not found. 
Available: ['diannibereval_source', 'diannibereval_bigbio_kb', 
            'diannibereval_en_source', 'diannibereval_en_bigbio_kb', 'diannibereval_es_source', 
            'diannibereval_es_bigbio_kb', 'diannibereval_bigbio_t2t']
```

Reply:
It seems that #330 does not fully fix this. Please, refer to #331 for a sequel :laughing: